ethpm_types/utils.py

```diff
@@ -1,9 +1,21 @@
 from enum import Enum
 from hashlib import md5, sha3_256, sha256
 
+from hexbytes import HexBytes as BaseHexBytes
+
 CONTENT_ADDRESSED_SCHEMES = {"ipfs"}
 
 
+class HexBytes(BaseHexBytes):
+    @classmethod
+    def __get_validators__(cls):
+        yield cls.validate
+
+    @classmethod
+    def validate(cls, v):
+        return HexBytes(v)
```

Comment:
Would this work?

```suggestion
        yield cls
```

Reply:
It doesn't, it then would return the actual class. `{'byte_bytes': <class 'ethpm_types.utils.HexBytes'>, ...}`

Reply:
Weird, I guess it does have to be a class method like `lambda cls, v: ...`